src/main/scala/kanaloa/reactive/dispatcher/metrics/MetricsCollector.scala

```diff
@@ -14,16 +15,16 @@ object NoOpMetricsCollector extends MetricsCollector {
 
 object MetricsCollector {
   /** Create a MetricsCollector from a root typesafe Config */
-  def fromRootConfig(pipelineName: String, config: Config = ConfigFactory.load())(implicit system: ActorSystem) = {
-    val path = "workPipeline.metrics"
+  def fromRootConfig(dispatcherName: String, config: Config = ConfigFactory.load())(implicit system: ActorSystem) = {
+    val path = "reactiveDispatcher.metrics"
```

Comment:
maybe have this root config namespace as `kanaloa` instead of `reactiveDispatcher`